src/actions/guides/testing/debugging_tests.cr

```diff
@@ -0,0 +1,72 @@
+class Guides::Testing::DebuggingTests < GuideAction
+  guide_route "/testing/debugging"
+
+  def self.title
+    "Debugging Tests"
+  end
+
+  def markdown : String
+    <<-MD
+    ## Introduction
+  
+    Determining why a given test is failing can sometimes be difficult. This guide serves as a jumping-off point for some tips and tricks that you can leverage to resolve these issues more quickly for your Lucky applications.
+
+    ## Lucky Flow
+
+    ### Taking Screenshots
+    
+    When you're running an integration spec that tests your application interface using Lucky Flow, sometimes getting a picture of what's going on at a certain point on the screen can be incredibly helpful when diagnosing a failing test.
+
+    Say that we have the following spec file:
+
+    ```crystal
+    require "../spec_helper"
+
+    describe "Authentication flow" do
+      it "works" do
+        flow = AuthenticationFlow.new("test@example.com")
+
+        flow.sign_up "password"
+        flow.should_be_signed_in
+        flow.sign_out
+        flow.sign_in "wrong-password"
+        flow.should_have_password_error
+        flow.sign_in "password"
+        flow.should_be_signed_in
+      end
+    end
+    ```
+
+    If we wanted to see what the state of the interface looks like after the `should_be_signed_in` step, all we need to do is insert a `take_screenshot` method call like so:
+
+    ```crystal
+    require "../spec_helper"
+
+    describe "Authentication flow" do
+      it "works" do
+        flow = AuthenticationFlow.new("test@example.com")
+
+        flow.sign_up "password"
+        flow.should_be_signed_in
+        flow.take_screenshot
```

Comment:
Maybe a space before and after this to quicker highlight where that change is being made?

Reply:
Added, and threw in a comment as well!